modin/core/execution/ray/common/engine_wrapper.py

```diff
@@ -307,6 +307,31 @@ def post_materialize(self, materialized):
         """
         raise NotImplementedError()
 
+    def __reduce__(self):
```

Comment:
Is this class intended to work in remote kernels?

Reply:
It seems this class is used in remote functions because we ues it in MetaList, which we use in remote functions.

Reply:
Seems some functions pass the non-materialized partition lengths to remote functions.